Projects/Team2/Vlad Borisovets/Animals/Animals/Program.cs

```diff
@@ -0,0 +1,212 @@
+	[TestFixture]
+	class Program
+	{
+		/* Опишіть клас Тварини, який містить 
+         поля: РікНародження, Колір
+         конструктор з параметрами
+         функції консольного введення та виведення полів
+         Getteres Setters
+         Метод GetAge() що виводить повні роки
+         overridden ToString() метод, який викликає метод Voice()
+         Реалізуйте клас наслідник Fish, який має додаткові поля Kind (морська\річкова), Species. Конструктор з параметрами, додаткові getters та setters. Overridden input() та output()
+         Створіть колекцію тварин, додати кілька різних тварин та риб до цієї колекції
+         реалізуйте виведення риб старіших ніж 3 роки
+         відсортуйте інформацію за видом (Species)
+         збережіть колекцію у файл
+         реалізуйте опрацювання виключень (Exceptions)
+         сереалізуйте (Serialize) колекцію в XML файл
+         десереалізуйте колекцію з XML файла
+         напишіть unit тести до реалізованих функцій*/
+		public static List<Animals> animals = new List<Animals>();
+		static void Main(string[] args)
+		{
+			//var animals = new List<Animals>();
+			animals.Add(new Animals(2001, "red"));
+			animals.Add(new Animals(2002, "green"));
+			animals.Add(new Animals(2003, "red"));
+			animals.Add(new Fish(2001, "red", "sea", "Amberjack"));
+			animals.Add(new Fish(2016, "White", "Mackerel", "Barracuda"));
+			animals.Add(new Fish(2014, "White", "Mackerel", "Barracuda"));
+			animals.Add(new Fish(2019, "White", "Mackerel", "Barracuda"));
+			animals.Add(new Fish(2008, "Black", "Perch", "Bass"));
+			animals.Add(new Fish(2014, "Black", "Perch", "Bass"));
+			animals.Add(new Fish(2020, "Black", "Perch", "Bass"));
+			mods();
+		}
+
+		public static void mods()
+		{
+			Console.WriteLine("Select mode:\n0 - Add new animal\n1 - Add new Fish\n2 - Show all animals\n3 - Show all Fish\n4 - Fish older than 3 years and sorted by species" +
+				"\n5 - Write to file 'Animals.txt'\n6 - Serialize and Deserialize list cars to 'Animals.xml'");
+			int mode = Int32.Parse(Console.ReadLine());
+			switch (mode)
+			{
+				case 0:
+					GettersAnimal();
+					mods();
+					break;
+				case 1:
+					GettersFish();
+					mods();
+					break;
+				case 2:
+					SettersAllAnimals();
+					mods();
+					break;
+				case 3:
+					SettersAllFish();
+					mods();
+					break;
+				case 4:
+					fishOld();
+					mods();
+					break;
+				case 5:
+					WrittenToFile(animals);
+					mods();
+					break;
+				case 6:
+					SerializeAndDeserialize(animals);
+					mods();
+					break;
+
+			}
+
+		}
+		//Add new Animal
+		public static void GettersAnimal()
+		{
+			Console.WriteLine("Write year of bith your animal: ");
+			int YearOfBirth = Int32.Parse(Console.ReadLine());
+			Console.WriteLine("Write color your animal: ");
+			string Color = Console.ReadLine();
+			animals.Add(new Animals(YearOfBirth, Color));
+		}
+		//Add new Fish
+		public static void GettersFish()
+		{
+			Console.WriteLine("Write year of bith fish: ");
```

Comment:
This method could cause errors. Please rewrite this by using try/catch or TryParse()